build_rules.gradle

```diff
@@ -432,3 +433,9 @@ ext.applyAvroNature = {
   println "applyAvroNature with " + (it ? "$it" : "default configuration") + " for project $project.name"
   apply plugin: "com.commercehub.gradle.plugin.avro"
 }
+
+// Apply the apt-eclipse plugin so the eclipse import can work
+apply plugin: 'eclipse'
```

Comment:
Note that your applying this to every project and not just java projects, is this what you intended to do?

Also, all global plugins are defined at the top and plugins explain why they are added like the description for `io.pry.gradle.offline_dependencies`